frontend/viewer/src/lib/entry-editor/object-editors/EntryEditor.svelte

```diff
@@ -118,6 +121,9 @@
       setTimeout(() => {
         const newEntityElem = editorElem?.querySelector('.highlight');
         if (newEntityElem) {
+          if (highlightedEntity?.autofocus && !IsMobile.value)
+            findFirstTabbable(newEntityElem?.querySelector('.editor-primitive'))?.focus();
```

Comment:
the tabbable library has an isFocusable check, should we maybe be using that instead considering we want to focus on something?